src/components/v2/Pagination/styles.ts

```diff
@@ -0,0 +1,42 @@
+import { css } from '@emotion/react';
+import { useTheme } from '@mui/material';
+
+export const useStyles = () => {
+  const theme = useTheme();
+  return {
+    root: css`
+      display: flex;
+    `,
+    button: css`
+      width: ${theme.spacing(8)};
+      height: ${theme.spacing(8)};
+      padding: 0;
+      background-color: ${theme.palette.background.paper};
+      color: ${theme.palette.text.secondary};
+      margin-left: ${theme.spacing(1)};
+      margin-right: ${theme.spacing(1)};
+      transition: color 0.3s;
+      border-radius: ${theme.shape.borderRadius.verySmall}px;
+
+      &:hover {
+        color: ${theme.palette.text.primary}!important;
+      }
+    `,
+    getButtonStyles: ({ isActive }: { isActive: boolean }) => css`
+      color: ${isActive ? theme.palette.text.primary : theme.palette.text.secondary};
+    `,
+    arrowButtonWrapper: css`
+      display: flex;
+      align-items: center;
+      line-height: 1px;
+    `,
+    iconReverted: css`
+      transform: rotate(180deg);
```

Comment:
Let's create a `leftArrow` icon instead, this way we make sure it respects the designs' dimensions and placement

Reply:
but that makes a one more file to load. the arrow in design is the same but rotated. why do you think the current realisation is not suitable?

Reply:
Ok I see. In some instances, the icon could be placed differently in the viewbox depending on the direction of the arrow. If you're sure the left arrow corresponds exactly to the right right with a 180 degree rotation, let's go for it.